test/Visualization_Tests/src/Widgets/Database_Widgets_Spec.enso

```diff
@@ -0,0 +1,43 @@
+from Standard.Base import all
```

Comment:
I'd suggest to create subdirectories here - like `Table`, `Database` etc. as I imagine this will grow to contain many more specs once we start adding widgets everywhere.